docs/parameter_files.rst

```diff
@@ -77,8 +84,21 @@ Time and frequency
             * (``end_freq``)
 
     As long as one of the sets from each category above is met by the supplied keywords, the frequency array will be successfully built.
-    Likewise for the time array:
+    You can also just give an explicit ``freq_array``.
+
+
+Time
+^^^^
 
+    The time array is specified similarly. The entries in the ``time_array`` indicate the **beginning of each time step in Julian date**. The ``integration_time`` is the time step size in seconds. The user may also specify ``duration_hours`` or ``duration_days`` to specify the total time covered by all time steps. The following relations among parameters hold::
```

Comment:
I find this confusing -- it sounds like the time_array values are for the start of the integration, not the center of the integration.

Reply:
That was my intention. Should it be the center of the integration?

Reply:
I think it should. UVData time_array values correspond to the center of the integration. Of course for pyuvsim, we aren't really integrating, we're just calculating the value at a moment in time. We might want to clarify that in the docs.

Reply:
Okay, that makes sense. I think in terms of the `simsetup.py` code, both standards are written the same. I'll update the docs.